src/routes/resource-edit-route/resource-edit-route.test.js

```diff
@@ -0,0 +1,333 @@
+    expect(mockGetAccessTypes).toHaveBeenCalled();
+  });
+
+  describe('when handle Cancel', () => {
+    it('should redirect to the view package page', () => {
+      const { getByRole } = renderResourceEditRoute();
+
+      fireEvent.click(getByRole('button', { name: 'ui-eholdings.label.icon.closeX' }));
+
+      expect(history.replace).toHaveBeenCalledWith({
+        pathname: `/eholdings/resources/${model.id}`,
+        search: location.search,
+        state: {
+          eholdings: true,
+        },
+      });
+    });
+  });
+
+  describe('when handle onSubmit', () => {
+    it('should render Custom labels accordion and updateResource', () => {
+      const { getByText, getByRole } = renderResourceEditRoute({
+        model: {
+          ...model,
+          isSelected: true,
+        },
+      });
+
+      fireEvent.submit(getByRole('button', { name: 'stripes-components.saveAndClose' }));
+
+      expect(getByText('ui-eholdings.resource.customLabels')).toBeDefined();
+      expect(mockUpdateResource).toHaveBeenCalled();
+    });
+  });
+
+  describe('when update request resolves', () => {
+    it('should redirect to the view title page', () => {
+      const { rerender } = renderResourceEditRoute({
+        model: {
+          ...model,
+          update: {
+            ...model.update,
+            isPending: true,
+          },
+        },
+      });
+
+      rerender(getResourceEditRoute());
+
+      expect(history.replace).toHaveBeenCalled();
+    });
+  });
+
+  describe('when id in url has changed', () => {
+    it('should handle getResource', () => {
+      const newId = 'new-test-id';
+
+      const { rerender } = renderResourceEditRoute();
+
+      rerender(getResourceEditRoute({
+        match: {
+          ...match,
+          params: {
+            id: newId,
+          },
+        },
+      }));
+
+      expect(mockGetResource).toHaveBeenCalledWith(newId);
+    });
+  });
+
+  describe('when package is destroyed', () => {
+    it('should redirect back to search page', () => {
+      const { rerender } = renderResourceEditRoute();
+
+      rerender(getResourceEditRoute({
+        model: {
+          ...model,
+          destroy: {
+            isResolved: true,
+            errors: [],
+          },
+          isLoaded: true,
+          isSelected: true,
+        },
+      }));
+
+      expect(history.replace).toHaveBeenCalled();
+    });
+  });
+
+  describe('when package is added to holdings', () => {
+    it('should update resource', () => {
+      const { getByText } = renderResourceEditRoute({
+        model: {
+          ...model,
+          isSelected: false,
+        },
+      });
+
+      fireEvent.click(getByText('ui-eholdings.addToHoldings'));
+
+      expect(mockUpdateResource).toHaveBeenCalledWith({
+        ...model,
+        isSelected: true,
+      });
+    });
+  });
+
+  describe('when a managed package is deselected', () => {
+    it('should call destroyResource', () => {
```

Comment:
You have here 'should call **destroyResource**' but in `expect` you check if mock**UpdateResource** has been called. Please recheck and change it accordingly.